integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/CoinBaseErrorDialog.kt

```diff
@@ -0,0 +1,94 @@
+package org.dash.wallet.integration.coinbase_integration.ui
+
+import android.content.Intent
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.net.Uri
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.view.Window
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import androidx.fragment.app.DialogFragment
+import org.dash.wallet.common.ui.viewBinding
+import org.dash.wallet.integration.coinbase_integration.R
+import org.dash.wallet.integration.coinbase_integration.databinding.DialogCoinbaseErrorBinding
+
+class CoinBaseErrorDialog : DialogFragment() {
```

Comment:
We have several dialog frameworks in our app.  Later it will be good to try to use one for all.  We also have `FancyAlertDialog` and `AlertDialog` and maybe some others.

Recently, @ClaudeHangui created a `BaseDialogFragment` which adds some functionality concerning autologout that will need to be incorporated as well.

Reply:
We will handle this issue in a different story.